<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Default test"
           tests="2"
           failures="2"
           errors="0"
           skipped="0"
           time="26.497">
  <properties />
            <testcase name="tearDown" time="0.000" classname="com.dbyl.tests.baiduEditor">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.dbyl.tests.baiduEditor.tearDown(baiduEditor.java:52)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
  org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:232)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="testJD" time="26.497" classname="com.dbyl.tests.baiduEditor">
                <failure type="org.openqa.selenium.remote.UnreachableBrowserException"
                  message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &apos;2.52.0&apos;, revision: &apos;4c2593c&apos;, time: &apos;2016-02-11 19:03:33&apos;
System info: host: &apos;localhost&apos;, ip: &apos;127.0.0.1&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.4&apos;, java.version: &apos;1.7.0_80&apos;
Driver info: driver.version: RemoteWebDriver"
              >
<![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.52.0', revision: '4c2593c', time: '2016-02-11 19:03:33'
System info: host: 'localhost', ip: '127.0.0.1', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.7.0_80'
Driver info: driver.version: RemoteWebDriver
  org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
  org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
  org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
  org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:158)
  com.dbyl.libarary.utils.DriverFactory.getRemoteDriver(DriverFactory.java:192)
  com.dbyl.tests.baiduEditor.testJD(baiduEditor.java:25)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
Caused by: java.net.SocketException: Network is unreachable
  java.net.PlainSocketImpl.socketConnect(Native Method)
  java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
  java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
  java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
  java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
  java.net.Socket.connect(Socket.java:579)
  org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
  org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
  org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
  org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
  org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
  org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
  org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
  org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
  org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
  org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
  org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
  org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
  org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
  org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
  org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
  org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
  org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
  org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:158)
  com.dbyl.libarary.utils.DriverFactory.getRemoteDriver(DriverFactory.java:192)
  com.dbyl.tests.baiduEditor.testJD(baiduEditor.java:25)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
